### DISCUSSION TITLE
Contains

### DISCUSSION THREAD

**Discussion by Masynchin** [coady/placeholder#23] [General]
How do I this?

~~~python
filter(_ in "123", names)
~~~

I tried to implement it since `_` has not `__contains__`, but the only result was not-working `"123" in _`. Hopes the author can implement this better than me.

**Comment by coady** [comment]
Unfortunately there's no way to support `__contains__`. Python coerces the result type of some `__dunder__` methods, so it's impossible to return the instance.
```python
In []: class C:
    ...:     def __contains__(self, value):
    ...:         return 'contains'
    ...: 

In []: None in C()
Out[]: True
```
